### PR TITLE
[teleop] Created a science logger gui

### PR DESCRIPTION
**Pull request by catkims**
GUI:
image

Menu:
image

The camera UI has not been finalized yet.

### CODE REVIEW

**Review by CameronTressler — APPROVED** [review]
Cameras will of course need updates but I think this is a good starting point

File: base_station/gui/src/components/GenerateLogger.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        this.csvFileData = [  
+        [timeString, spectral_data.d0_1,spectral_data.d1_1,spectral_data.d2_1],  
+        [timeString, spectral_data.d0_2,spectral_data.d1_2,spectral_data.d2_2],  
+        [timeString, spectral_data.d0_3,spectral_data.d1_3,spectral_data.d2_3],  
+        [timeString, spectral_data.d0_4,spectral_data.d1_4,spectral_data.d2_4,],  
+        [timeString, spectral_data.d0_5,spectral_data.d1_5,spectral_data.d2_5,],  
+        [timeString, spectral_data.d0_6,spectral_data.d1_6,spectral_data.d2_6,]
+        ];  
+        //define the heading for each row of the data  
+        var csv = 'Timestamp,Spectral 0,Spectral 1, Spectral 2\n';
```
Comment (cgiger00) [inline base_station/gui/src/components/GenerateLogger.vue:36]: should traid data also be logged?

**Comment by cgiger00** [comment]
can this be renamed?

**Comment by cgiger00** [comment]
new pr with ish gui to be opened soon

**Pull request closed without merging**